flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -500,6 +493,7 @@ public void testGetKeysAndNamespaces() throws Exception {
                                     keysByNamespace.computeIfAbsent(entry.f1, k -> new HashSet<>());
                             assertTrue("Duplicate key for namespace", keys.add(entry.f0));
                         });
+                assertThat(keysByNamespace.size(), is(namespaces.length));
```

Comment:
```suggestion
                assertEquals("Unexpected size of keys and namespaces", namespaces.length, keysByNamespace.size());
```

Reply:
This test will be different from the others in the test. I think it's gonna look weird

Reply:
On Flink [website](https://flink.apache.org/how-to-contribute/code-style-and-quality-common/) we can find this
"And instead use:"

```
assertThat(list)
    .hasSize(10)
    .allMatch(item -> item.length() < 10);
```

Reply:
What you mentioned is a good way for Junit 5 which tells us not to assert in the for loop for collection.
What you added in Junit 4 is not a same way, just as the website said, 'Don’t use Hamcrest'. 
And it's a deprecated method.